app/src/main/java/ch/epfl/qedit/frontendBackendInterface/Status.java

```diff
@@ -0,0 +1,36 @@
+package ch.epfl.qedit.frontendBackendInterface;
+
+public class Status {
+    private boolean ok;
+    private String message;
+
+    public static final String okMessage = "The status is OK";
+
+    public Status() {
+        ok = true;
+        message = okMessage;
+    }
+
+    public Status(String errorMessage) {
+        ok = false;
+        message = errorMessage;
+    }
+
+    public boolean isOk() {
+        return ok;
+    }
+
+    public String getMessage() {
+        return message;
+    }
+
+    public void error(String message) {
```

Comment:
Maybe we could make it a static factory method ?

Reply:
We could build statuses like this
`Status status = Status.ok();`
`Status status = Status.error("Something went wrong");`

Reply:
I implemented this behavior.